root/view/src/main/java/at/fhv/itb5c/view/tournament/TournamentModel.java

```diff
@@ -0,0 +1,111 @@
+package at.fhv.itb5c.view.tournament;
+
+import java.rmi.RemoteException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import at.fhv.itb5c.commons.dto.rmi.IMatchRMI;
+import at.fhv.itb5c.commons.dto.rmi.ITeamRMI;
+import at.fhv.itb5c.commons.dto.rmi.ITournamentRMI;
+import at.fhv.itb5c.logging.ILogger;
+import at.fhv.itb5c.rmi.client.RMIClient;
+import at.fhv.itb5c.view.AppState;
+import javafx.beans.property.DoubleProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.Property;
+import javafx.beans.property.SimpleDoubleProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+public class TournamentModel implements ILogger {
+	private StringProperty _tournamentName;
+	private ObjectProperty<LocalDate> _date;
+	private DoubleProperty _fee;
+	private Property<ObservableList<String>> _guestTeams;
+	private Property<ObservableList<ITeamRMI>> _homeTeams;
+	private Property<ObservableList<Object>> _teams;
+	private Property<ObservableList<IMatchRMI>> _matches;
+	private Long _departmentId;
+
+	private ITournamentRMI _rmiTournament;
+	
+	public TournamentModel(){
```

Comment:
Shouldn't the constructor maybe enforce setting of _rmiTournament? (--> setITournamentRMI(ITournamentRMI))